packages/create-turbo/src/index.ts

```diff
@@ -172,11 +177,6 @@ async function run() {
       ...sharedPkg[pkgKey],
       ...projectPkg[pkgKey],
     };
-    for (let key in sharedPkg[pkgKey]) {
-      if (sharedPkg[pkgKey][key] === "*") {
```

Comment:
@jaredpalmer I don't believe we need this block anymore, any packages with `*` are local, and we can leave packages with `latest` as is to pull up to date versions on install.